auto/modules/php

```diff
@@ -149,7 +149,11 @@ nxt_feature_test="
     #include <php_main.h>
 
     int main() {
+    #if PHP_VERSION_ID < 80200
```

Comment:
The check should be in brackets (to match Unit's coding style). I.e

```diff
#if (PHP_VERSION_ID < 80200)
```

Reply:
brackets added (despite there is some other places without ex https://github.com/nginx/unit/blob/master/src/nxt_php_sapi.c#L19)